mix.exs

```diff
@@ -22,6 +27,63 @@ defmodule Wanda.MixProject do
     ]
   end
 
+  defp docs do
+    [
+      main: "readme",
+      logo: "priv/static/images/trento.svg",
+      extra_section: "GUIDES",
+      source_url: @source_url,
+      assets: "guides/assets/",
+      extras: extras(),
+      groups_for_extras: groups_for_extras(),
+      groups_for_modules: groups_for_modules(),
+      nest_modules_by_prefix: [
+        Wanda.Catalog,
+        Wanda.Executions,
+        Wanda.Messaging,
+        WandaWeb
+      ]
+    ]
+  end
+
+  defp extras() do
+    [
+      "README.md",
+      "CHANGELOG.md",
+      "CONTRIBUTING.md",
+      "guides/specification.md",
+      "guides/expression_language.md",
+      "guides/gatherers.md"
+    ]
+  end
+
+  defp groups_for_extras do
+    %{
```

Comment:
It looks strange that we use a map here and a list of keywords in the `groups_for_modules` XD (i see the 2nd in most of the projects)